.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,13 @@ Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 - [ ] documentation is changed or added
 - [ ] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)
 
+<!--
+Please also uncheck this box if you would like other contributors not to fix up
+minor changes such as style issues, typos, etc. with the PR for you,
+rather than asking you to address these by yourself.
+-->
+
+- [ ] I would prefer to address minor change requests myself
```

Comment:
The box is unchecked by default and therefore the comment above is not really corresponding correctly. It should probably be checked by default and then the comment above fits again.